CMakeLists.txt

```diff
@@ -192,6 +193,21 @@ if(AMR_WIND_ENABLE_HELICS)
   target_compile_definitions(${amr_wind_lib_name} PRIVATE AMR_WIND_USE_HELICS)
 endif()
 
+if(AMR_WIND_TEST_WITH_PYTHON)
+  set(CMAKE_PREFIX_PATH ${PYTHON_DIR} ${CMAKE_PREFIX_PATH})
+  find_package(Python3 REQUIRED COMPONENTS Interpreter)
+  # Developer note: modify this list if modifying scripts used in python-based regression tests
+  set(PYTHON_REQUIREMENTS numpy netCDF4 matplotlib pathlib pandas)
+  foreach(pymod IN LISTS PYTHON_REQUIREMENTS)
```

Comment:
could we `venv` this stuff? I would not want to mess with the user's python modules. To be honest, we should "package" the python scripts up properly and then maybe we can use that?

Reply:
This is meant to check for the required modules, it doesn't do any installation or setup.

Reply:
ah sorry I missed the 'show"